Eludia/Presentation/Skins/XLSX.pm

```diff
@@ -748,13 +749,17 @@ sub draw_table_header_cell {
 	}
 
 	my $i = 1;
-	while ($i <= $rowspan) {
+	while ($i < $rowspan) {
 		if (!$worksheet -> {__map_str} -> {$row + $i}) {
 			$worksheet -> {__map_str} -> {$row + $i} = [];
 		}
-		for (my $j = 0; $j < $colspan; $j++) {
-			my $k = $i == $rowspan ? 0 : 1;
-			push @{$worksheet -> {__map_str} -> {$row + $i}}, $k;
+		for (my $j = 0; $j <= $col + $colspan - 1; $j++) {
+
+			next if $worksheet -> {__map_str} -> {$row + $i} [$j];
+
+			my $k = $j < $col ? $worksheet -> {__map_str} -> {$row + $i} [$j]  + 0 : 1;

```

Comment:
range of `$j`: `0..$_REQUEST {__xl_col} + $colspan - 1`
range of $k: `smth ... 1 ... smth`
is `$j < $col` correct for `$colspan > 1`?